src/views/virtualmachines/creation-wizard-new/steps/InstanceTypesSteps/ComputeResourcesStep/components/SelectInstanceTypeSection/components/RedHatProvidedInstanceTypesSection/components/InstanceTypeSizeMenu/InstanceTypeSizeMenu.tsx

```
@@ -3,32 +3,46 @@ import React, { FC } from 'react';
+                selectedInstanceType: selectedSeries ? `${selectedSeries}.${size}` : size,
+              });
+              setValue(CREATE_VM_FORM_FIELDS_INSTANCE_TYPE_DATA.SELECTED_INSTANCE_TYPE, {
+                name: selectedSeries ? `${selectedSeries}.${size}` : size,
+                namespace: null,
+              });
+            }}
+            selectedSize={value}
+            seriesName={selectedSeries}
+            sizes={instanceTypeSizes}
+          />
+        )}
```

ESLint

[error] Expected "render" to come before "name"

(perfectionist/sort-jsx-props)